src/stress.jl

```diff
@@ -50,35 +50,35 @@ Reference:
 """
 module Stress
 
-using GeometryTypes, Compat, FixedSizeArrays
+using GeometryTypes, Compat, StaticArrays
 import Base: start, next, done, *
 
-function (*){T<:LinAlg.BlasFloat,S<:FixedArray}(A::StridedMatrix{T}, x::StridedVector{S})
+function (*){T<:LinAlg.BlasFloat,S<:StaticArray}(A::StridedMatrix{T}, x::StridedVector{S})
     A_mul_B!(similar(x, S, size(A,1)), A, x)
 end
 
-immutable Layout{M1<:AbstractMatrix, M2<:AbstractMatrix, VP<:AbstractVector,FT<:AbstractFloat}
+immutable Layout{M1<:AbstractMatrix, M2<:AbstractMatrix, VP<:AbstractVector, FT<:AbstractFloat}
     δ::M1
     weights::M2
     positions::VP
-    pinvLw::Matrix{FT}
+    pinvLw::Matrix{Float64}
     iterations::Int
     abstols::FT
     reltols::FT
     abstolx::FT
 end
 
 
-function initialweights(D, T=eltype(D))
+function initialweights(D)
     map(D) do d
-        x = T(d^(-2.0))
-        isfinite(x) ? x : zero(T)
+        x = Float64(d^(-2.0))
```

Comment:
Sorry, I have absolutely no time to care about this package right now. 
If you can't figure it out, I guess that's then just how it is.